examples/dataset/README.md

```diff
@@ -2,16 +2,18 @@
 
 This directory contains example Kolena Datasets integrations for different machine learning workflows.
 
-- [Binary Classification](./classification): open-source classification models tested using the
-  [Dogs vs. Cats](https://www.kaggle.com/c/dogs-vs-cats) dataset
+- [Classification](./classification): open-source classification models tested using the
```

Comment:
What's the ordering logic?

Reply:
Looks like alphabetical by top level "ML workflow"

Reply:
Ack

Reply:
Yes, to match Github's folder ordering